core/usecase/get-comments.usecase.go

```diff
@@ -0,0 +1,23 @@
+package usecase
+
+import (
+	"github.com/nhaancs/GrabGoTrainingWeek5Assignment/core/repository"
+)
+
+// GetCommentsUsecase usecase
+type GetCommentsUsecase struct {
+	Repo repository.CommentRepository
+}
+
+// NewGetCommentsUsecase func
+func NewGetCommentsUsecase(repo repository.CommentRepository) *GetCommentsUsecase {
+	return &GetCommentsUsecase {
+		Repo: repo,
+	}
+}
+
+// Execute GetCommentsUsecase
+func (usecase *GetCommentsUsecase) Execute(params... interface{}) (interface{}, error) {
```

Comment:
We don't need `params...` here
Nitpick about naming again, this function should be differently, execute express toward a function just do something and return error